src/plugins/overlay/overlay.js

```diff
@@ -115,6 +115,7 @@ var componentName = "wb-overlay",
 
 		$overlay
 			.addClass( "open" )
+			.attr( "role", "dialog" )
```

Comment:
Can you try to wrap it with the dialog element?

Reply:
it is much easier to manipulate the role attribute here. You will have multiple dialogs in the page. unless you do wrap and unwrap all the time.

Reply:
Ideally we will use dialog, but let do that change later. I will create a new issue to get that fixed eventually.